dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -35,78 +35,83 @@ def get_failed_job_count
     Delayed::Job.where.not(failed_at: nil).count
   end
 
+  def get_my_failed_job_count
+    Delayed::Job.where.not(failed_at: nil).where('handler LIKE ?', "%job_class: #{self.class.name}%").count
+  end
```

Comment:
LGTM for now, though this could stop working if we set `destroyed_failed_jobs = true` as discussed in slack.